app/extensions/edm/__init__.py

```diff
@@ -183,7 +199,9 @@ def request_passthrough_parsed(
         return response, response_data, result_data
 
     # Provides the same validation and exception raising as above but just returns the result
-    def request_passthrough_result(self, tag, method, passthrough_kwargs, args=None):
+    def request_passthrough_result(
+        self, tag, method, passthrough_kwargs, args=None, request_headers=None
```

Comment:
shouldn't you be passing request_headers to the request_passthrough_parsed below?

Reply:
absolutely yes.